docs/hw/display/nv3/pramdac.rst

```diff
@@ -18,12 +18,16 @@ Introduction
 MMIO registers
 ==============
 
-.. space:: 8 pramdac 0x1000 RAMDAC, video overlay, cursor, and PLL control
+.. space:: 32 pramdac 0x1000 RAMDAC, video overlay, cursor, and PLL control
```

Comment:
8 is correct here - it's the byte size in bits, which is always 8 for all MMIO directly accessible from PCI. The only usecases for other space widths are spaces where addressing happens in units different than normal bytes (eg. index-data reg pairs where data is not 8-bit wide).